apps/src/templates/GameButtons.jsx

```diff
@@ -16,9 +24,13 @@ var GameButtons = function (props) {
         <img src="/blockly/media/1x1.gif" className="reset26"/>
       </button>
       {" " /* Explicitly insert whitespace so that this behaves like our ejs file*/}
-      <CompletionButton/>
+      {props.children}
```

Comment:
Nope, though right now props.children is never more than length 1. I'll have to be cognizant of this on some apps (like bounce) where we can have a bunch of buttons, but I think I'll still be okay because they're not created in an array.